voice/src/main/kotlin/gateway/Event.kt

```diff
@@ -0,0 +1,133 @@
+@file:OptIn(KordVoice::class)
+
+package dev.kord.voice.gateway
+
+import dev.kord.common.annotation.KordVoice
+import dev.kord.gateway.Gateway
+import dev.kord.gateway.start
+import dev.kord.voice.EncryptionMode
+import kotlinx.serialization.DeserializationStrategy
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.builtins.nullable
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.descriptors.buildClassSerialDescriptor
+import kotlinx.serialization.encoding.CompositeDecoder
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.json.JsonElement
+import mu.KotlinLogging
+
+private val jsonLogger = KotlinLogging.logger { }
+
+sealed class Event {
```

Comment:
rename this to something other than `Event`, something like `VoiceEvent` since we already have that in the core module and may cause confusion